app/Http/Controllers/CustomAuthController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Session;
+
+class CustomAuthController extends Controller
+{
+    public function login()
+    {
+        return view("auth.login");
+    }
+
+    public function registration()
+    {
+        return view("auth.registration");
+    }
+
+    public function registerUser(Request $request)
+    {
+        $request->validate([
+            'name' => 'required',
+            'email' => 'required|email|unique:users',
+            'password' => 'required|min:5|max:12'
+        ]);
+        $user = new User();
+        $user->naam = $request->name;
```

Comment:
Code, en variabel namen, in het engels. 
$user->name in plaats van $user->naam
$user->password in plaats van $user->wachtwoord